projects/core/src/auth/models/token-types.model.ts

```diff
@@ -15,4 +15,12 @@ export interface ClientToken {
   scope: string;
 }
 
-export type AuthenticationToken = UserToken | ClientToken;
+export interface OpenIdToken {
```

Comment:
Can we simplify this file to maybe two interfaces or extending from a common one?

Reply:
Good catch, we could